src/app/shell/header/header-error/header-error.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="header header-error container">
+  <div class="mid-header">
+    <div class="logo-wrapper d-none d-md-block">
+      <a
+        rel="home"
+        href="/home"
```

Comment:
This is not working in a prod like deployment with multi-channel basehref configuration